EssentialApp/EssentialApp/SceneDelegate.swift

```diff
@@ -39,6 +39,11 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 		LocalFeedImageDataLoader(store: store)
 	}()
 	
+	private func makeRemoteFeedImageCommentsLoader(feedImage: FeedImage) -> RemoteFeedImageCommentsLoader {
+		let url = baseURL.appendingPathComponent("image/\(feedImage.id.uuidString)/comments").absoluteURL
```

Comment:
`absoluteURL` is not needed - remove it.

```suggestion
		let url = baseURL.appendingPathComponent("image/\(feedImage.id.uuidString)/comments")
```